code/__DEFINES/dcs/signals.dm

```diff
@@ -432,6 +436,8 @@
 #define COMSIG_TURF_ON_SHUTTLE_MOVE "turf_on_shuttle_move"
 ///from /turf/open/temperature_expose(datum/gas_mixture/air, exposed_temperature)
 #define COMSIG_TURF_EXPOSE "turf_expose"
+///from /datum/thrownthing/find_target(): (datum/thrownthing, actual_target)
+#define COMSIG_TURF_FIND_THROWNTHING_TARGET "turf_find_thrownthing_target"
```

Comment:
I really hate this naming scheme you've got going on here

Reply:
I'm not good at naming things. Got any suggestion?